text/lang-detection/model.jl

```diff
@@ -61,67 +61,79 @@ function get_processed_data(args)
     args.alphabet_len = length(alphabet)
 
     ## See which chars will be represented as "unknown"
-    unique(filter(x -> x ∉ alphabet, join(vcat(values(corpora)...))))
-
+    unkChars = unique(filter(x -> x ∉ alphabet, join(vcat(values(corpora)...))))
+    
     dataset = [(onehotbatch(s, alphabet, '_'), onehot(l, langs)) for l in langs for s in corpora[l]] |> shuffle
 
     train, test = dataset[1:end-args.test_len], dataset[end-args.test_len+1:end]
-    return train, test
+    (testX, testY) = unzip(test)
+    return train, testX, testY, langs
 end
 
+# https://stackoverflow.com/questions/36367482/unzip-an-array-of-tuples-in-julia
+unzip(a) = (getfield.(a, x) for x in fieldnames(eltype(a)))
+
```

Comment:
```suggestion
```
Removed in favour of suggestion in next comment (on `(testX, testY) = unzip(test)` line).